src/bank.rs

```diff
@@ -39,13 +39,16 @@ use window::WINDOW_SIZE;
 /// but requires clients to update its `last_id` more frequently. Raising the value
 /// lengthens the time a client must wait to be certain a missing transaction will
 /// not be processed by the network.
-pub const MAX_ENTRY_IDS: usize = 1024 * 16;
+pub const MAX_ENTRY_IDS: usize = 1024 * 32;
 
 pub const VERIFY_BLOCK_SIZE: usize = 16;
 
 /// Reasons a transaction might be rejected.
 #[derive(Debug, PartialEq, Eq, Clone)]
 pub enum BankError {
+    /// This Pubkey is being processed in another transaction
+    AccountInProgress,
```

Comment:
Something called `Account` can't be "in progress." Consider following bind()'s error and call it `AccountInUse`.